### PR TITLE
 Improve Avocado jobs command

### LINKED ISSUE #5761: Improve Avocado jobs command
**Describe the bug**

1. `avocado jobs list` returns unsorted list of job results. One improvement is to sort the list by `mtime`. This help find the recent jobs. 
2. `avocado jobs show` shows weird end time because we don't have the `end` attribute in the `results.json` any more. 

I have a fix on the first issue, but looking for inputs for the second one. One feasible fix is to replace the end time with the start time. 

**Steps to reproduce**
Steps to reproduce the behavior with a reproducible whenever possible.

```
# run a few tests
avocado run /bin/true
avocado run /bin/true
avocado run /bin/true
# 1. list
avocado jobs list
# 2. show
avocado jobs show
```

**Expected behavior**

1. 
```
b36e709ed01af2567e4df81842b678fb9dca51eb 2023-09-08 03:04:52.701753   1 (1/0/0/0)
11861a8dd228a86e7c0f2b3583fe6ca7011af99c 2023-09-08 03:00:01.887965   1 (1/0/0/0)
6a29261e8d23e963270c42c9147781fed3ca6118 2023-09-08 02:55:27.117886   1 (1/0/0/0)
```

2. 

```
JOB ID      : latest

Test ID     End Time            Run Time Status
1-/bin/true <correct time> 0.012107 PASS
```

**Current behavior**

1. 
```
b36e709ed01af2567e4df81842b678fb9dca51eb 2023-09-08 03:04:52.701753   1 (1/0/0/0)
6a29261e8d23e963270c42c9147781fed3ca6118 2023-09-08 02:55:27.117886   1 (1/0/0/0)
11861a8dd228a86e7c0f2b3583fe6ca7011af99c 2023-09-08 03:00:01.887965   1 (1/0/0/0)
```

2.
```
JOB ID      : latest

Test ID     End Time            Run Time Status
1-/bin/true 1970/01/01 01:51:03 01:51:03 0.012107 PASS
```

**System information (please complete the following information):**
 - OS: `Ubuntu 18.04.6 LTS`
 - Avocado version: `Avocado 102.0`
 - Avocado installation method: reproduce on github & pip installation

### PR DESCRIPTION
**Pull request by v1siuol**
Fix #5761

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,6 +22,9 @@ docs/source/api/utils
 docs/source/api/plugins
 docs/source/api/optional-plugins
 docs/source/guides/writer/libs/data/vmimage
+.venv
```
Comment (richtja) [inline .gitignore:25]: Please remove those changes from your commit, this is not related to avocado-framework and it should be part of your [local .gitignore](https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b)

**Review by richtja — CHANGES_REQUESTED** [review]
Hi @v1siuol thank you for your contribution, I very appreciate it. Overall, the changes LGTM. I have just some comments to style, so our checks will be ok. 

Please add your sign into your commit based on our [Commit style guide](https://avocado-framework.readthedocs.io/en/latest/guides/contributor/chapters/styleguides.html#commit-style-guide).

Run `black .` command in avocado root dir, that will fix all your style issues based on our style requirements.

If you need any help with those changes, don't hesitate to ask. We will help you.

File: avocado/plugins/jobs.py [hunk 2]
```diff
@@ -153,7 +157,7 @@ def handle_show_command(self, config):
         # We could improve this soon with more data and colors
         self._print_job_details(data)
         LOG_UI.info("")
-        self._print_job_tests(results_data.get("tests"))
+        self._print_job_tests(results_data)
```
Comment (v1siuol) [inline avocado/plugins/jobs.py:160]: The thing here is both `start` and `end` inside the `tests` are monotonic time. So I have to take the start local timestamp from one layer above.

File: .gitignore (hunk 1 above)
Comment (v1siuol) [reply .gitignore]: Oh I see, fixed

File: avocado/plugins/jobs.py (hunk 2 above)
Comment (richtja) [reply avocado/plugins/jobs.py:160]: Yes, you are right here. I have looked more deeply into this problem and find out that we gather the `start` and `end` with `time.time()` in seconds, but we do not store it to the results file. I have opened PR #5768, which should help you with this when it is merged.

Comment (v1siuol) [reply avocado/plugins/jobs.py:160]: Awesome, yeah that would be super helpful. Btw, I realize I should add the sign-off by "-s". I will fix them later.

**Review by richtja — APPROVED** [review]
Hi @v1siuol, thank you for your update, it LGTM.
